proxy/config_test.go

```diff
@@ -76,4 +83,21 @@ func TestWhenHasErgoFile(t *testing.T) {
 			tt.Errorf("Expected service to be added")
 		}
 	})
+
+	t.Run("It returns an error if there's an invalid declaration", func(tt *testing.T) {
+		fileContent, err := ioutil.ReadFile("../.ergo")
+
+		if err != nil {
+			tt.Skipf("Could not load initial .ergo file")
```

Comment:
I consider using Skip vs Skipf when you do not have variable substitution a best practice.